plugin/pkg/scheduler/algorithm/predicates/utils_test.go

```diff
@@ -68,3 +71,116 @@ func ExampleFindLabelsInSet() {
 	// label1=value1,label2=value2,label3=will_see_this
 	// pod1,pod2,
 }
+
+func TestFindLabelsInSet(t *testing.T) {
+
+	nsPod := v1.Pod{
```

Comment:
Why do you use a pod to set labels? Can't we just use `Labels := map[string]string{...}` instead?